### DISCUSSION TITLE
Copilot Chat is too dumb right now

### DISCUSSION THREAD

**Discussion by JohnKirk1337** [community/community#118470] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I've been trying Copilot and I'm disappointed by the performance of the chat feature. Recently Bing's Copilot AI has been dumbed down to hell. It won't do things it once did easily. It confuses questions unless you're very specific. It gives wrong or shallow answers. It forgets previous instructions. 

But I thought Github's copilot wouldn't suffer from these nerfs, because unlike Bing's Copilot, GitHub's is a paid product. But it did. Github Copilot's Chat will avoid writing long code by saying "you can do it by..." even if you explicitly asked him to write it. It'll sometimes just leave "//code goes here" comments instead of actually writing the method. I've seen Bing's Copilot do these exact same things.

**Comment by jfullstackdev** [comment]
AI is still far from understanding the full content of what we are trying to do, even the hype of the latest Devin AI is still far from us, and that's why it will not replace humans totally doing the job

but what we can get the outmost is the way it guides us in coding, if you know how to use it as a tool, then yes, you feel like you matured 5 years in coding suddenly, that's what I felt, it can even answer very specific questions that StackOverflow may not answer or you need to wait for it to be answered since Copilot is trained itself on GitHub Data

so again, as for me, it's a very useful tool and Copilot Chat is more than enough for me, even my company requires us to use that always as it increases efficiency

**Comment by Eptagone** [comment]
Exactly. And also, Copilot was much smarter when I started using it, but now it's too dumb and constantly forgets what we were talking about.

**Reply by poo3610** [reply]
i am also facing the same issue, what should i do

**Reply by MusicMakerOwO** [reply]
That's the token limit, all AI has that issue if you talk long enough, but as long as you aren't sending multiple paragraphs or huge code blocks then you should be fine for most conversations

**Reply by angeloorru** [reply]
Same issue for me. I stopped the subscription. It ain't worth paying for nothing

**Reply by xkuza** [reply]
> i am also facing the same issue, what should i do

ditch the b-ch. openai never forgets what we talk about.

**Reply by Apurva2551** [reply]
Exactly. I noticed the same thing. Copilot used to feel much smarter when I first started using it, but now it constantly forgets what we were talking about and gives shallow answers. It’s frustrating because it feels like a downgrade, not an improvement.

**Comment by way27** [comment]
It seems like everyone has this problem

Copilot has no context, and the previous answer was in Chinese, but suddenly answered me in English later, which is really annoying.
<details>
<summary>Android Studio screentshot</summary>

![image](https://github.com/user-attachments/assets/236cf1c5-41c0-434a-8985-79a3c23eb697)
</details>

**Comment by eret9616** [comment]
really stupid now i cant use copilot chat

**Reply by maciejcieslik** [reply]
I agree that Copilot is not able to do the simplest things. It should be for free as open beta now...

**Comment by AllyTechEngineering** [comment]
Just today, I was using MaterialStateProperty for iconButtonTheme and was getting a warning that it was deprecated. I used GitHub Copilot and it said that it was not deprecated. The VS Code pop up window suggested changing to WidgetStateProperty and the warning went a way. I copied and pasted the code in the GitHub Copilot chat and it said that WidgetStateProperty was not a Flutter class (it is: https://api.flutter.dev/flutter/widgets/WidgetStateProperty-class.html) and that I must have created it. I am going to ask for refund for the monthly fee they are charging for such garbage. I pay a monthly for chatGpt and I can upload my code as files and it works pretty good.

**Comment by weathon** [comment]
it makes simple mistakes and cannot understand prompt.

I asked it to extract movement intensity from a rgb intensity map, it first extract it in RGB space, and then I said you should do it in hsv space, then it takes the V as intensity, which is wrong, it should be the S. I asked it to fix it, and then it made up and being firm that intensity = norm(H, S). Those are completely wrong. This was on GPT-4o, which is no where near this stupid.
